src/Apps/Order/Components/PaymentPicker.tsx

```diff
@@ -199,12 +203,18 @@ export class PaymentPicker extends React.Component<
 
     const orderCard = this.props.order.creditCard
 
-    let defaultCard
+    const creditCardsArray: any = []
 
-    if (creditCardSelection.type === "new") {
-      defaultCard = "new"
-    } else {
-      defaultCard = orderCard ? orderCard.id : creditCardSelection.id
+    creditCards.edges.map(e => {
```

Comment:
You could do this when initializing the array:

`const creditCardsArray = creditCards.edges.map(e => e.node)`

That way it also retains the type information instead of making it an `any`